openmdao/core/explicitcomponent.py

```diff
@@ -120,6 +135,21 @@ def _setup_partials(self):
         """
         Call setup_partials in components.
         """
+        # if self.options['derivs_method'] in ('cs', 'fd'):
```

Comment:
Is this block of coded waiting for a future PR?

Reply:
No I moved that block into Component.  It's cleaned up now.